### PR TITLE
Add Bing to caches.json

### PR DESCRIPTION
**Pull request by ssantosms**
Bing cache is missing from caches.json

Addresses issue #18992

### CODE REVIEW

**Comment by aghassemi** [comment]
Thanks @ssantosms. To @Gregable for review and merge.

**Comment by ssantosms** [comment]
@Gregable could you please review this change?

File: caches.json [hunk 1]
```diff
@@ -15,6 +15,14 @@
       "cacheDomain": "cdn.cloudflare.com",
       "updateCacheApiDomainSuffix": "amp.cloudflare.com",
       "thirdPartyFrameDomainSuffix": "cloudflareamp.net"
+    },
+    {
+      "id": "bing",
+      "name": "Bing AMP Cache",
+      "docs": "https://www.bing.com/",
```
Comment (Gregable) [inline caches.json]: Do you have a better documentation URL?

Maybe https://www.bing.com/webmaster/help/bing-amp-cache-bc1c884c ?

Comment (ssantosms) [reply caches.json]: I do. And I thought I had it there. My bad. I will update it

Comment (ssantosms) [reply caches.json]: Done

**Comment by ssantosms** [comment]
@Gregable the change you requested was addressed. Please take a look at your convenience

**Pull request merged**